lib/Model.js

```diff
@@ -539,6 +547,11 @@ Model.update = function(NewModel, key, update, options, next) {
     options.updateTimestamps = true;
   }
 
+  // default updateExpires to false
+  if (typeof options.updateExpires === 'undefined') {
+    options.updateExpires = false;
```

Comment:
Once again I don't think we need this section. 'undefined' is considered falsey so I don't think we need this part since the default will be undefined or false.